testing/trino-testing-minio/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>380-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-testing-minio</artifactId>
```

Comment:
Does it need to be separate?
Can it go into `trino-testing-containers`?

Reply:
> Does it need to be separate?

The MinIO client is used both in product tests and in `trino-delta-lake` project.
So it should be in a common place to avoid duplicating its code in both projects.


> Can it go into trino-testing-containers?
Adding the client logic to `trino-testing-containers` is not a good fit because this is utility is not a _container_.

I am assuming that you are hinting to avoid adding yet another module to the Trino project.
I investigated the option of adding the class/module to `tempto` but haven't found there a good fit.
I also investigated the other projects under `testing/` directory, but also haven't found a good fit.

Finally I saw `trino-testing-kafka` and decided to create `trino-testing-minio`.
Alternative solutions are welcome.

Reply:
> Adding the client logic to `trino-testing-containers` is not a good fit because this is utility is not a _container_.

I agree, it is not a container. However, it aids working with a container. 
The `trino-testing-containers` already contains MinIO-related code.